external/radiation/tests/test_config.py

```diff
@@ -0,0 +1,35 @@
+from radiation.config import RadiationConfig
+import pytest
```

Comment:
This is a good idea but I'm holding off for now because the radiation driver and wrapper are not set up in a way that allows for easy mocking/testing. Some refactors to disentangle the I/O from the code we'd want to test will be coming in a future PR I think.